src/Components/AddPaletteForm/AddPaletteForm.js

```diff
@@ -22,7 +22,7 @@ const AddPaletteForm = (props) => {
   return (
     <section className="palettes">
       <div>
-        <button onClick={props.assignToRandomColor}>New Colors</button>
+        <button className="new-color" onClick={props.assignToRandomColor}>New Colors</button>
```

Comment:
It looks like this has been broken into separate components, this is way better.